__tests__/lib/createGHA.test.ts

```diff
@@ -191,7 +210,7 @@ describe('#createGHA', () => {
         delete process.env.TEST_RDME_NPM_SCRIPT;
       });
 
-      it('should not run if repo only contains non-GitHub remotes', () => {
+      it('should not run if repo solely contains non-GitHub remotes', () => {
```

Comment:
Renamed this so I don't have to see it when grepping the testbed for `.only` lol